web-common/src/features/dashboards/time-dimension-details/TDDHeader.svelte

```diff
@@ -112,21 +113,47 @@
   function startPivotForTDD() {
     const pivot = $dashboardStore?.pivot;
 
-    if (pivot.rows.length || pivot.columns.length) {
+    if (
+      pivot.rows.dimension.length ||
+      pivot.columns.measure.length ||
+      pivot.columns.dimension.length
+    ) {
       showReplacePivotModal = true;
     } else {
       createPivot();
     }
   }
+
+  // This needs to be reworked - bgh
```

Comment:
Is this a stale comment?

Reply:
Not at the time. Looks like you reworked it in https://github.com/rilldata/rill/pull/4002